{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "5d2e6c11_2fc7bb43",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 21
      },
      "lineNbr": 0,
      "author": {
        "id": 1000341
      },
      "writtenOn": "2023-10-19T09:00:47Z",
      "side": 1,
      "message": "Triggering CI as failure didn\u0027t seem relevant.",
      "revId": "2c1ff744e567a28e1742a9bb1e8ad89d5ad0592e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}